data/paracosma-nepal.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Paracosma Nepal",
+    "establishedAt": 1451589361,
+    "location": "naxal, Kathmandu",
+    "website": "https://www.paracosma.com/",
+    "email": null,
```

Comment:
Email according to : https://www.rojgarisanjal.com/company/detail/paracosma-470 : `info@paracosma.com`